README.md

```diff
@@ -168,6 +168,17 @@ Entering interactive mode (type "help" for commands, "o" for options)
 (pprof) top
 ```
 
+## Exporter
+If you are not satisfied with the build-in web monitor in Locust, you can run prometheus_exporter.py instead of dummy.py as your master.
+
+Try this
+
+```bash
+locust --master -f prometheus_exporter.py
+```
+
+Thanks to Prometheus and Grafana, you will get an awesome dashboard: [Locust for Prometheus](https://grafana.com/grafana/dashboards/12081)
```

Comment:
We are missing a guidance about configuring Prometheus to monitor the Locust master.

Reply:
I have written a post introduce this, but it is in Chinese.

[locust-with-prometheus-and-grafana](https://bugvanisher.github.io/2020/04/05/locust-with-prometheus-and-grafana/)